src/components/victory-group/victory-group.js

```diff
@@ -359,7 +375,8 @@ export default class VictoryGroup extends React.Component {
     if (role !== "group-wrapper" && role !== "stack-wrapper") {
       return undefined;
     }
-    return child.props.colorScale || props.colorScale;
+    return props.theme ? child.props.colorScale || props.colorScale || props.theme.props.colorScale
```

Comment:
code smell: too many things going on in this ternary. Maybe extract `child.props.colorScale || props.colorScale` into a variable first.